s3api/handlers_buckets.go

```diff
@@ -13,198 +14,134 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+func executeRollBack(t *[]func() error) {
+	if t != nil {
+		tasks := *t
+		log.Errorf("executing rollback of %d tasks", len(tasks))
+		for i := len(tasks) - 1; i >= 0; i-- {
```

Comment:
Can we print what we are actually doing for the rollback? Maybe the method name, e.g. `DeleteEmptyBucket`. Not sure if that's possible.

Reply:
the functions that get executed will log what they are doing.  something like:

```
[s3api-6c86d547f4-dbw8f s3api] time="2019-02-19T21:03:38Z" level=error msg="Conflict: EntityAlreadyExists: EntityAlreadyExists: A policy called camdensawsomespacebucket0123-BktAdmPlc already exists. Duplicate names are not allowed.\n\tstatus code: 409, request id: d68bf0ff-3489-11e9-a0b0-dfe339fc73f7 (EntityAlreadyExists: A policy called camdensawsomespacebucket0123-BktAdmPlc already exists. Duplicate names are not allowed.\n\tstatus code: 409, request id: d68bf0ff-3489-11e9-a0b0-dfe339fc73f7)"
[s3api-6c86d547f4-dbw8f s3api] time="2019-02-19T21:03:38Z" level=error msg="recovering from panic: failed to create policy: Conflict: EntityAlreadyExists: EntityAlreadyExists: A policy called camdensawsomespacebucket0123-BktAdmPlc already exists. Duplicate names are not allowed.\n\tstatus code: 409, request id: d68bf0ff-3489-11e9-a0b0-dfe339fc73f7 (EntityAlreadyExists: A policy called camdensawsomespacebucket0123-BktAdmPlc already exists. Duplicate names are not allowed.\n\tstatus code: 409, request id: d68bf0ff-3489-11e9-a0b0-dfe339fc73f7)"
[s3api-6c86d547f4-dbw8f s3api] time="2019-02-19T21:03:38Z" level=error msg="executing rollback of 1 tasks"
[s3api-6c86d547f4-dbw8f s3api] time="2019-02-19T21:03:38Z" level=info msg="deleting bucket: camdensawsomespacebucket0123"
```